apps/bfd-model/bfd-model-rif-samples/src/test/java/gov/cms/bfd/model/rif/samples/SampleDataColumnsTest.java

```diff
@@ -42,8 +44,8 @@ public void logEnumColumns() {
    * gov.cms.bfd.model.rif.samples.StaticRifResourceGroup#SAMPLE_A} data file headers.
    */
   @Test
-  public void verifySampleAColumns() {
-    verifyColumns(StaticRifResourceGroup.SAMPLE_A);
+  public void verifySampleBColumns() {
+    verifyColumns(StaticRifResourceGroup.SAMPLE_B);
```

Comment:
I left the sample A files alone for now so we don't mess with the existing tests. Sample B contains the new xref columns which we need to use for this test since we're checking that all columns in the file are in the schema.